framework/resources/Functional/no_result_set/DRILL_6834_select_query.sql

```diff
@@ -0,0 +1,3 @@
+set `exec.return_result_set_for_ddl` = false;
+select first_name, last_name from cp.`employee.json` order by employee_id limit 5;
```

Comment:
This is just a valid usecase right? The option is meant for DDLs. But sure, good to test if this works for all SQL statements.

Reply:
Yes, you're right. We need to be sure that if the option is false, regular SQL queries will return result.